lib/config/fileconf.go

```diff
@@ -613,6 +614,40 @@ func (u *UniversalSecondFactor) Parse() (services.U2F, error) {
 	return res, nil
 }
 
+// AllowTCPForwarding describes the possible values of the
+// `allow_tcp_forwarding` SSH settings key. Essentially a wrapper around
+// regular.SSHPortForwardingMode to plug it into the YAML parser.
+type AllowTCPForwarding regular.SSHPortForwardingMode
+
+// UnmarshalYAML parses a YAML representation of a SSHPortForwardingMode,
+// failing if the value cannot be converted.
+func (mode *AllowTCPForwarding) UnmarshalYAML(unmarshal func(interface{}) error) error {
+	var text string
+	err := unmarshal(&text)
+	if err != nil {
+		return trace.Wrap(err)
+	}
+
+	var value regular.SSHPortForwardingMode
+	switch strings.ToLower(text) {
+	case "yes", "all":
+		value = regular.SSHPortForwardingModeAll
+	case "no":
+		value = regular.SSHPortForwardingModeNone
+	case "local":
+		value = regular.SSHPortForwardingModeLocal
+	default:
```

Comment:
What does it default to if not set in the config? Maybe add a comment here, otherwise it looks like config parsing will fail if it's not set.

Reply:
If it's not set this will never be invoked and default to the zero value of `SSHPortForwardingModeAll`. Will add more  commentary to clarify.

Reply:
Done